integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/mock/MockCloudProvider.java

```diff
@@ -84,7 +82,7 @@ public StackTestDtoBase stack(StackTestDtoBase stack) {
 
     @Override
     public DistroXTestDtoBase distrox(DistroXTestDtoBase distrox) {
-        return distrox.withGatewayPort(getSparkServerPort(distrox.getTestContext()));
+        return distrox.withGatewayPort(10090);
```

Comment:
why fixed port ?

Reply:
Our new service has a dedicated port but in under the "path shifting or unique path" task, this will be removed